src/__tests__/handleAction-test.js

```diff
@@ -195,4 +195,39 @@ describe('handleAction()', () => {
         .to.deep.equal({ number: 3 });
     });
   });
+
+  describe('with invalid actions', () => {
+    it('should throw a descriptive error when the action object is missing', () => {
+      const action1 = createAction('ACTION_1');
+      const reducer = handleAction(
+        action1,
+        (state) => state
```

Comment:
For this and the tests below:

- can we use the identity function from lodash?
- can we inline the call to `createAction` (`action1` is not used elsewhere)?

Also, unless there's a good reason to disable`max-len`, let's stick to it. Since we're testing an exact match, can we use the string directly?